app/controllers/idv/link_sent_controller.rb

```diff
@@ -85,8 +85,7 @@ def take_photo_with_phone_successful?
 
     def document_capture_session_result
       @document_capture_session_result ||= begin
```

Comment:
Wonder if we need the `begin` block anymore at this point? Seems like this could be flattened.

```rb
      @document_capture_session_result ||= document_capture_session&.load_result
```